lib/releases.sh

```diff
@@ -2,29 +2,49 @@
 
 function get_latest_release() {
   set +x
+  # get last page
   if [ -z "${GITHUB_TOKEN:-}" ]; then
-    release="$(curl -sL "${1}")" || ( set -x && exit 1 )
+    last_page=$(curl -s -I "${1}" -H "Authorization: token $GITHUB_TOKEN" | grep '^link:' | sed -e 's/^link:.*page=//g' -e 's/>.*$//g') || (set -x && exit 1)
```

Comment:
`(set -x && exit 1)` creates a subshell, the parenthesis does that, so the `exit 1` exits only that subshell, and not the script. If you want to perform two actions, use curly braces, `{set -x && exit 1}`.

Also, you want to assign the subshell output of `$( ... )` with quotes, so do `last_page="$( ... )"`

Reply:
`${set -x ...}` is definitely wrong, but what does it complain about `{ set -x ... }` ?

Reply:
thanks! done